server/config.yml

```diff
@@ -0,0 +1,7 @@
+inference:
```

Comment:
love this, much better than flooding the CLI args

Reply:
Thank you!